builder/scaleway/config.go

```diff
@@ -116,51 +114,39 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 
 	c.UserAgent = useragent.String()
 
-	// Deprecated variables
-	if c.Organization == "" {
-		if os.Getenv("SCALEWAY_ORGANIZATION") != "" {
-			c.Organization = os.Getenv("SCALEWAY_ORGANIZATION")
-		} else {
-			log.Printf("Deprecation warning: Use SCALEWAY_ORGANIZATION environment variable and organization_id argument instead of api_access_key argument and SCALEWAY_API_ACCESS_KEY environment variable.")
-			c.Organization = os.Getenv("SCALEWAY_API_ACCESS_KEY")
-		}
-	}
-	if c.Organization != "" {
-		warnings = append(warnings, "organization_id is deprecated in favor of project_id")
-		c.ProjectID = c.Organization
-	}
-
-	if c.Token == "" {
-		c.Token = os.Getenv("SCALEWAY_API_TOKEN")
+	configFile, err := scw.LoadConfig()
```

Comment:
What have the higher precedence? IMHO it should be the env vars

Reply:
The environment will override the configuration file